app/src/main/kotlin/com/saltedge/authenticator/features/main/MainActivityViewModel.kt

```diff
@@ -192,4 +201,9 @@ class MainActivityViewModel(
         appContext.applyPreferenceLocale()
         onRestartActivityEvent.postUnitEvent()
     }
+
+    private fun wipeApplication() {
+        interactorV1.wipeApplication()
+        interactorV2.wipeApplication()
```

Comment:
redundant call . `interactorV1` have wiped application before. 
You can merge both interactors into one.